zellij-server/src/screen.rs

```diff
@@ -1981,6 +1999,39 @@ pub(crate) fn screen_thread_main(
                     screen.render()?;
                 }
             },
+            ScreenInstruction::GoToTabName(tab_name, create, client_id) => {
+                let client_id = if client_id.is_none() {
+                    None
+                } else if screen
+                    .active_tab_indices
+                    .contains_key(&client_id.expect("This is checked above"))
+                {
+                    client_id
+                } else {
+                    screen.active_tab_indices.keys().next().copied()
+                };
+                if let Some(client_id) = client_id {
+                    if let Ok(is_exist) = screen.go_to_tab_name(tab_name.clone(), client_id) {
```

Comment:
Let's change `is_exist` to `tab_exists`, ok?

Reply:
Sure